### DISCUSSION TITLE
Switching between NVENC and AMF encoder

### DISCUSSION THREAD

**Discussion by pyerix** [alvr-org/ALVR#1719] [Q&A] [answered]
Hello, i have a laptop with an AMD cpu that comes with integrated graphics (Lucienne), but the laptop has a dedicated Nvidia GPU (3050 Ti) for heavy rendering and 3D graphics, ALVR defaults to dgpu rendering and uses the same gpu for encoding. It's possible to switch from the Nvidia encoder to the AMD encoder?

If not this could be a feature to implement.

Thanks!!!

**Comment by zmerp** [comment, marked as answer]
Yes, this is planned.

**Reply by pyerix** [reply]
:+1:
